### PR TITLE
You can no longer sentience tourist bots

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
fix: You can no longer sentience tourist bots
fix: Sentient robots are no longer invincible
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/robot_customer.dm [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ///Robot customers
 /mob/living/simple_animal/robot_customer
 	name = "space-tourist bot"
-	maxHealth = 50000000 //go fuck yourself
-	health = 50000000
+	maxHealth = 1000 //go fuck yourself
+	health = 1000
+	status_flags = GODMODE //:trollface:
```
Comment (Sparkezel) [inline code/modules/mob/living/simple_animal/friendly/robot_customer.dm:6]:
```suggestion
```
Dear god just remove it

File: code/modules/mob/living/simple_animal/friendly/robot_customer.dm [hunk 2]
```diff
@@ -83,3 +85,7 @@
 		. += "<span class='notice'>Their order was: \"[attending_venue.order_food_line(ai_controller.blackboard[BB_CUSTOMER_CURRENT_ORDER])].\"</span>"
 
 
+
+/mob/living/simple_animal/robot_customer/Login()
```
Comment (Sparkezel) [inline code/modules/mob/living/simple_animal/friendly/robot_customer.dm:89]:
```suggestion
/mob/living/simple_animal/robot_customer/Login()
```
Why does that even exist....

**Comment by Sparkezel** [comment]
Better than what armhulen did though

**Comment by anugla** [comment]
giving the tourist bots sentience is fun thoguh...

File: code/modules/mob/living/simple_animal/friendly/robot_customer.dm (hunk 2 above)
Comment (Qustinnus) [reply code/modules/mob/living/simple_animal/friendly/robot_customer.dm:89]: ????

File: code/modules/mob/living/simple_animal/friendly/robot_customer.dm [hunk 3]
```diff
@@ -83,3 +85,7 @@
...
+	. = ..()
+	status_flags &= ~GODMODE //owned admin
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/friendly/robot_customer.dm:91]: if an admin wants to make a robot customer sapient, they probably want it to remain invincible anyway (and can remove the godmode flag if they don't)

**Comment by ATH1909** [comment]
what did you THINK would happen, floyd

File: code/modules/mob/living/simple_animal/friendly/robot_customer.dm [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ///Robot customers
 /mob/living/simple_animal/robot_customer
 	name = "space-tourist bot"
-	maxHealth = 50000000 //go fuck yourself
-	health = 50000000
+	maxHealth = 1000 //go fuck yourself
+	health = 1000
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/friendly/robot_customer.dm:5]:
```suggestion
	maxHealth = 100
	health = 100
```

Comment (Pumpkinoe) [reply code/modules/mob/living/simple_animal/friendly/robot_customer.dm:5]: then won't they just get murdered for shits and giggles, which is annoying for people trying to actually use them

Comment (ATH1909) [reply code/modules/mob/living/simple_animal/friendly/robot_customer.dm:5]: I think that the unusually large amount of health would actually encourage people to try to murder them, but hey, maybe I'm wrong.

If someone intentionally murders your tourist bots, I think that counts as them sabotaging your workplace, which should be a valid reason to dunk on them.

Comment (TheBonded) [reply code/modules/mob/living/simple_animal/friendly/robot_customer.dm:5]: They respawn at least one per minute

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**